crawl-ref/source/wiz-item.cc

```diff
@@ -1173,9 +1175,17 @@ static void _debug_acquirement_stats(FILE *ostat)
             "cloud immunity",
 #endif
             "harm",
+            "shadows",
             "rampaging",
             "infusion",
+            "light",
+            "rage",
+            "mayhem",
+            "guile",
+            "energy",
+            "debug_randart",
```

Comment:
item-prop-enum.h's list of egos ends with NUM_REAL_SPECIAL_ARMOURS and then NUM_SPECIAL_ARMOURS. Not sure what the real_special is for so put a placeholder here for that one.

Reply:
REAL_SPECIAL is meant for the end of the egos that can spawn. I'll tweak this when I merge it.